multimedia/templates/multimedia/base_list.html

```diff
@@ -7,7 +7,7 @@
 <h2>{{ multimedia_type }} List</h2>
 <div class="container">
   <ul class="list-unstyled">
-    {% for item in multimedia %}
+    {% for item in object_list %}
```

Comment:
You are going to break the API for the rest of the views here. I suggest doing this in a separate PR, or you should fix the other views if they are merged first